plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/RemoteDockerNode.java

```diff
@@ -104,14 +104,19 @@ public void bindWorkspace() throws MachineException {
                                                                          .withDetach(false));
             final List<String> ownerIds = new ArrayList<>(4);
             final ValueHolder<String> error = new ValueHolder<>();
-            dockerConnector.startExec(StartExecParams.create(exec.getId()), message -> {
-                if (message.getType() == LogMessage.Type.STDOUT) {
-                    ownerIds.add(message.getContent());
+            String errorMessage = "Can't detect container user ids to chown backed up workspace " + workspaceId + " files.";
+            dockerConnector.startExec(StartExecParams.create(exec.getId()), logMessage -> {
+                if (logMessage.getType() == LogMessage.Type.STDOUT) {
+                    ownerIds.add(logMessage.getContent());
                 } else {
-                    LOG.error("Can't detect container user ids to chown backed up workspace " + workspaceId + "files. " + message);
-                    error.set("Can't detect container user ids to chown backed up workspace " + workspaceId + "files");
+                    LOG.error(errorMessage + " " + logMessage);
+                    error.set(errorMessage);
                 }
             });
+            if (ownerIds.size() < 2 && error.get() == null) {
+                LOG.error(errorMessage  + (ownerIds.size() == 1 ? " " + ownerIds.get(0) : ""));
+                error.set(errorMessage);
```

Comment:
This error in not informative. Please add explanation to error what has happened.